osx-arm64: missing qcollectiongenerator

trying to build pyside2-feedstock locally for osx_arm64 I run into this issue:

```
-- optional module Qt5XmlPatterns found (). Looked in: $PREFIX/lib/cmake/Qt5XmlPatterns
CMake Error at /Users/lo/mambaforge/conda-bld/pyside2_1652084189920/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_place/lib/cmake/Qt5Help/Qt5HelpConfig.cmake:14 (message):
  The imported target "Qt5::Help" references the file

     "/Users/lo/mambaforge/conda-bld/pyside2_1652084189920/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_place/bin/qcollectiongenerator"

  but this file does not exist.  Possible reasons include:

  * The file was deleted, renamed, or moved to another location.

  * An install or uninstall procedure did not complete successfully.

  * The installation package was faulty and contained
```

--- a/.ci_support/migrations/gstreamer120.yaml
+++ b/.ci_support/migrations/gstreamer120.yaml
@@ -1,7 +1,0 @@
-__migrator:
-  build_number: 1
-  kind: version
-  migration_number: 1
-gstreamer:
-- '1.20'
-migrator_ts: 1644012262.9946759

--- a/recipe/build.sh
+++ b/recipe/build.sh
@@ -110,6 +110,7 @@ if [[ $(uname) == "Linux" ]]; then
                 -shared \
                 -nomake examples \
                 -nomake tests \
+                -make tools \
                 -verbose \
                 -skip wayland \
                 -skip qtwebengine \

--- a/recipe/build.sh
+++ b/recipe/build.sh
@@ -217,6 +217,7 @@ if [[ ${HOST} =~ .*darwin.* ]]; then
                 -shared \
                 -nomake examples \
                 -nomake tests \
+                -make tools \
                 -verbose \
                 -skip wayland \
                 -skip qtwebengine \

--- a/recipe/meta.yaml
+++ b/recipe/meta.yaml
@@ -45,7 +45,7 @@ source:
     folder: opengl32sw                                                                                            # [win64]
 
 build:
-  number: 0
+  number: 1
   detect_binary_files_with_prefix: true
   run_exports:
     - {{ pin_subpackage('qt-main', max_pin='x.x') }}

